mod.ts

```diff
@@ -1,11 +1,15 @@
 import {
   i32,
-  SizedFFIType,
+  PackedStruct,
```

Comment:
I would suggest using the `AlignedStruct` type which should match the wgsl [Structure Member Layout specification](https://www.w3.org/TR/WGSL/#structure-member-layout).